### ISSUE TITLE
Focus problem

### ISSUE THREAD

**Issue by stile17** [DavidWeiss2/windows-Credential-Provider-library#5]
I have a rather strange case.

In my project I only implement ICredentialProviderCredential (without ICredentialProviderCredential2) to get my own tile. For now, I am creating a prototype so I have not implemented any functionalities under this tile and I noticed a problem that cancels the use of CP in this way in my company.

When my smartcard initializes (on the usb pen, Sign-In options appears) the field in which I enter the login or password loses focus and then the focus gets the first field to edit, which causes the password to be entered in the login field or the password begins to be re-entered . I'm talking about a standard password provider here, I don't click on the tile I added.

I found an entry in Microsoft's documentation that probably describes the problem accurately: https://docs.microsoft.com/en-us/windows/win32/api/credentialprovider/nn-credentialprovider-icredentialprovidercredentialevents2
but I don't call CredentialsChange in my code.

Can I get rid of this focus problem somehow?

**Comment by dumblob** [comment]
@stile17 did you manage to solve this issue somehow?

**Comment by stile17** [comment]
@dumblob no, we unfortunately had to deploy our solution with this problem :(
